config/confd_env_backend/templates/cloudfront-pk.pem.tmpl

```diff
@@ -0,0 +1 @@
+{{if exists "REGISTRY2_S3_CLOUDFRONT_PRIVATEKEY"}}{{getv "REGISTRY2_S3_CLOUDFRONT_PRIVATEKEY"}}{{end}}
```

Comment:
`{{getv` -> `{{getenv`

You can also remove the `if` and just do:

```
{{getenv "REGISTRY2_S3_CLOUDFRONT_PRIVATEKEY" ""}}
```

Reply:
Good spot! Thanks!